src/jdk.zipfs/share/classes/module-info.java

```diff
@@ -45,6 +45,9 @@
  *   <li>Open an existing file as a Zip file system</li>
  * </ul>
  *
+ * @implNote The Zip File System will throw a ZipException when opening an
+ * existing Zip file that contains Zip entries with "." or ".." in its name elements.
```

Comment:
Hello Lance, reading this sentence adds a bit of confusion since it uses the word "contains". Had I not known the implemenation details, this sentence would have made me think zip file with name elements of the form `foo.bar` or `hello..`  would also be rejected since these name elements "contain" `.` or `..`

Do you think we should change the word to something like "The Zip File System will throw a ZipException when opening an existing Zip file that has "." or ".." named entries"?

Reply:
I think the `@implNote` tag can be dropped here. For the text then maybe it could be tweaked to something like "The ZIP file system provider does not support opening an existing ZIP that contains entries with ...".

Reply:
Updated the note removing `@impleNote` thought about pre-pending **Note:** to make the wording stick out but held off for now. Updated the Release note accordingly

Jaikiran,  I think the use of name elements is fine as we use similar phrasing in `Path::getName()`